plugins/org.jkiss.dbeaver.model.sql/src/org/jkiss/dbeaver/model/sql/translate/SQLQueryTranslator.java

```diff
@@ -225,7 +225,8 @@ protected List<? extends SQLScriptElement> translateStatement(
             if (defChanged) {
                 String newQueryText = SQLFormatUtils.formatSQL(null,
                     sqlTranslateContext.getSyntaxManager(),
-                    createTable.toString());
+                    createTable.toString(),
+                    null);
```

Comment:
You don't need to call the overloaded method here. It will be called anyways because the original method calls the overloaded.